### PR TITLE
docs(weekly): add this week in databend 117

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

- Closes #issue

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/13516)

### CODE REVIEW

File: website/blog/2023-10-29-databend-weekly-117.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- [Docs | Monitor - Jaeger](https://databend.rs/doc/monitor/tools/jaeger)
+- [PR #13381 | feat: migrate tracing reporter from jaeger agent to opentelemetry collector (otlp)](https://github.com/datafuselabs/databend/pull/13381/)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Added new aggregate function `quantile_tdigest_weighted`.
+- Recluster supports distributed execution.
+- `DELETE` supports table aliases.
```
Comment (soyeric128) [inline website/blog/2023-10-29-databend-weekly-117.md:86]: Can u share the pull request for this feature?

File: website/blog/2023-10-29-databend-weekly-117.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's New
+
+Stay informed about the latest features of Databend.
+
+### Feature Preview: Read-Only ATTACH TABLE
```
Comment (soyeric128) [inline website/blog/2023-10-29-databend-weekly-117.md:41]: The feature is out now, as well as the document updates:) 
So I think we can get rid of "Feature Preview" from the title.

Comment (soyeric128) [inline website/blog/2023-10-29-databend-weekly-117.md:41]:
```suggestion
Read-Only ATTACH TABLE
```

File: website/blog/2023-10-29-databend-weekly-117.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+Maintaining large resources for just a few large-scale queries can bring high maintenance costs for enterprises. How can we mix local and cloud workloads, leverage Databend Cloud's instant scaling while ensuring query reliability, and further reduce the count of private instances to save on maintenance costs?
+
+Databend recently introduced **Read-only `ATTACH TABLE`**, allowing users with locally deployed Databend to share table data with Databend Cloud in read-only mode:
```
Comment (soyeric128) [inline website/blog/2023-10-29-databend-weekly-117.md:45]:
```suggestion
Databend recently introduced the READ_ONLY argument to `ATTACH TABLE`**, allowing users with locally deployed Databend to share table data with Databend Cloud in read-only mode:
```

File: website/blog/2023-10-29-databend-weekly-117.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's New
+
+Stay informed about the latest features of Databend.
+
+### Feature Preview: Read-Only ATTACH TABLE
+
+Maintaining large resources for just a few large-scale queries can bring high maintenance costs for enterprises. How can we mix local and cloud workloads, leverage Databend Cloud's instant scaling while ensuring query reliability, and further reduce the count of private instances to save on maintenance costs?
```
Comment (soyeric128) [inline website/blog/2023-10-29-databend-weekly-117.md:43]: Maintaining extensive resources for just a few large-scale queries can lead to high maintenance costs for enterprises. How about we combine local and cloud workloads, harness Databend Cloud's instant scaling computing capabilities, all while avoiding the additional storage costs associated with cloud usage?

File: website/blog/2023-10-29-databend-weekly-117.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+Databend recently introduced **Read-only `ATTACH TABLE`**, allowing users with locally deployed Databend to share table data with Databend Cloud in read-only mode:
+
+- All table data is shared in read-only mode.
+- Changes made to the table, including data and schema changes, will be performed and tracked on the user's local platform.
```
Comment (soyeric128) [inline website/blog/2023-10-29-databend-weekly-117.md:48]:
```suggestion
- Changes made to a table, including data and schema changes, will be performed and tracked on the user's local platform.
```

Comment (soyeric128) [reply website/blog/2023-10-29-databend-weekly-117.md:48]: I guess it is also true for non-read-only mode.

File: website/blog/2023-10-29-databend-weekly-117.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+```sql
+ATTACH TABLE <target_table_name> '<source-table-data-URI>' CONNECTION=(<connection_parameters>) READ_ONLY
+```
+
+Adding the `READ_ONLY` parameter to the `ATTACH TABLE` statement will map the table data and schema using the latest snapshot hint file stored at the specified source table data URI, and the target table can only be accessed in a read-only manner.
```
Comment (soyeric128) [inline website/blog/2023-10-29-databend-weekly-117.md:54]: 即使不是read-only, 也是用最新的snapshot

File: website/blog/2023-10-29-databend-weekly-117.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Support for Defining CONNECTION
+
+To more elegantly handle the mapping between role access control models in compute cluster instances and storage services, Databend plans to provide the ability to define and manage `CONNECTION`s.
+
+**CONNECTION** can hold identity credentials and access management (IAM) entities generated by external storage services, providing convenient and consistent access control experiences. Introducing **CONNECTION** can also simplify complex configurations in creating `STAGE` etc.
```
Comment (soyeric128) [inline website/blog/2023-10-29-databend-weekly-117.md:98]:
```suggestion
**CONNECTION** can hold identity credentials and access management (IAM) entities generated by external storage services, providing convenient and consistent access control experiences. Introducing **CONNECTION** can also simplify complex statements that includes a connection to object storages, such as creating an external stage, etc.
```

File: website/blog/2023-10-29-databend-weekly-117.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+The referenced syntax specification is as follows:
```
Comment (soyeric128) [inline website/blog/2023-10-29-databend-weekly-117.md:100]:
```suggestion

```

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**